server/datastore/mysql/hosts.go

```diff
@@ -763,9 +764,46 @@ func queryStatsToScheduledQueryStats(queriesStats []fleet.QueryStats, packName s
 	return scheduledQueriesStats
 }
 
+func winHostConnectedToFleetCond(in string) string {
```

Comment:
hm, I think I disagree here 🤔 . `IN` matches whatever is passed, as you mention this can be a reference to a column or an explicit value, how is this different from all other functions where we do something similar?

Reply:
In other places where we're giving the caller a chance to provide values, don't we try to follow the pattern `statement, args, err := applySomeFiler(...)`? 

I think what threw me is that it's not immediately clear where those responsibilities lie with this function. For example, who is responsible for ensuring that literal values are quoted properly (i.e. single quotes `'`)? 

Do we really mean to enable a caller to insert any SQL here? It just felt more open-ended than our other SQL generating functions.

Reply:
that's for the filters, right? but take a look, most of the functions that generate sql don't do any validation of the caller arguments.

having that said, I went wit this although I'm not fully convinced!

Reply:
> most of the functions that generate sql don't do any validation of the caller arguments.

That's probably true. I don't know if I'm looking for validations (which might be nice-to-haves) as much as clear signals to future callers via typing the function params (like we do for MDM statuses) and/or naming of params (e.g., `hostUUIDs []string` which combined with a return type of `statement, args, err` would make it clear who is responsible for what). 

That said, we can go ahead with this approach and just keep the conversation going more generally :)